rootless.h

```diff
@@ -1,11 +1,11 @@
-#ifdef THEOS_PACKAGE_INSTALL_PREFIX
+#ifdef XINA_SUPPORT // Only define this for rootful compilations that need support for xina
+#define ROOT_PATH(cPath) [[NSFileManager defaultManager] fileExistsAtPath:@"/var/LIY"] ? "/var/jb" cPath : cPath
+#define ROOT_PATH_NS(path) [[NSFileManager defaultManager] fileExistsAtPath:@"/var/LIY"] ? @"/var/jb" path : path
+#define ROOT_PATH_NS_VAR [[NSFileManager defaultManager] fileExistsAtPath:@"/var/LIY"] ? [@"/var/jb" stringByAppendingPathComponent:path] : path
+#define ROOT_PATH_VAR(cPath)(ROOT_PATH_NS([NSString stringWithUTF8String:cPath]).fileSystemRepresentation)
```

Comment:
This looks like it will cause a memory leak when compiling without ARC (`-fobj-arc`).

Reply:
Yeah but again, there is no other way to do it. This at least doesn't leak memory when arc is enabled, other solutions would always leak memory.